pandas/core/generic.py

```diff
@@ -3770,10 +3773,11 @@ def as_matrix(self, columns=None):
         --------
         pandas.DataFrame.values
         """
+        warnings.warn("method ``as_matrix`` will be removed in a future version. "
```

Comment:
"method method .as_matrix()...."
"Use .values instead"

Reply:
I not sure I understand, assume you want the backticks gone. Change uploaded.